examples/multiple_viewer_widget.py

```diff
@@ -0,0 +1,434 @@
+"""
+This is example how to have more than one viewer in the same napari instance.
+Additional viewers state will be synchronized with the main viewer and do not
+"""
+
+from copy import deepcopy
+
+import numpy as np
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import (
+    QCheckBox,
+    QDoubleSpinBox,
+    QPushButton,
+    QSplitter,
+    QTabWidget,
+    QVBoxLayout,
+    QWidget,
+)
+from superqt.utils import qthrottled
+from packaging.version import parse as parse_version
+
+import napari
+from napari.components.layerlist import Extent
+from napari.components.viewer_model import ViewerModel
+from napari.layers import Image, Labels, Layer, Vectors
+from napari.qt import QtViewer
+from napari.utils.action_manager import action_manager
+from napari.utils.events.event import WarningEmitter
+from napari.utils.notifications import show_info
+
+NAPARI_GE_4_16 = parse_version(napari.__version__) > parse_version("0.4.16")
+
+def copy_layer_le_4_16(layer: Layer, name: str = ""):
+    res_layer = deepcopy(layer)
+    # this deepcopy is not optimal for layers and images layers
```

Comment:
```suggestion
    # this deepcopy is not optimal for labels and images layers
```
Do you meant labels ?

Reply:
yes